docs/naps/2-conda-based-packaging.md

```diff
@@ -0,0 +1,557 @@
+    * [#4221](https://github.com/napari/napari/pull/4221) (Adjust conditions that trigger signing)
+    * [#4307](https://github.com/napari/napari/pull/4307) (Test installers in CI)
+    * [#4309](https://github.com/napari/napari/pull/4309) (Use conda-forge/napari-feedstock)
+    * [#4387](https://github.com/napari/napari/pull/4387) (Fix unlink errors on cleanup)
+    * [#4444](https://github.com/napari/napari/pull/4444) (Add versioning to installer itself)
+    * [#4447](https://github.com/napari/napari/pull/4447) (Use custom `.condarc` file)
+    * [#4525](https://github.com/napari/napari/pull/4525) (Revert to napari-versioned default paths)
+
+
+Adding the missing pieces to `constructor` involves changes in four different projects:
+
+* `conda/constructor` itself, the tool that builds the installer
+* `conda/menuinst`, responsible of creating the shortcuts
+* `conda/conda`, which relies on `menuinst` to delegate the shortcut creation, but this code path
+  was only enabled on Windows so far
+* `conda-forge/conda-standalone-feedstock`, which freezes a `conda` install along with their
+  dependencies and adds a thin layer on top to communicate with the `constructor` installer during
+  the installation process.
+
+The full list of PRs is available on
+[this issue comment](https://github.com/napari/napari/issues/4248#issuecomment-1066574137).
+A complete list of features can be found in the packaging documentation [^napari-packaging-docs].
+
+### Milestone 3: Adding support for conda packages in the plugin manager
+
+Support for conda/mamba handling of plugin installations was implemented in a base PR and then
+extended with different PRs:
+
+* Initial PR: [#2943](https://github.com/napari/napari/pull/2943) (Add initial support to install
+  plugins with conda/mamba)
+* Other PRs that improved and extended the functionality:
+    * [#3288](https://github.com/napari/napari/pull/3288) (Fix plugin updates)
+    * [#3369](https://github.com/napari/napari/pull/3369) (Add cancel actions to plugin manager)
+    * [#4074](https://github.com/napari/napari/pull/4074) (Use napari hub to list plugins)
+    * [#4520](https://github.com/napari/napari/pull/4520) (Rework how subprocesses are launched)
+
+Adding the necessary functionality to the plugin dialog, including:
+
+* Adapting the current `Installer` class to support not only `pip` but also `conda/mamba` in a
+  single code base.
+* Adding support for `install`, `uninstall`, and `update` commands.
+* Populating the listing with data obtained from the napari hub API [^napari-hub-api].
+* Updating the listing to filter and mark plugins based on their availability on conda-forge.
+
+Some UX/UI improvements that are not part of this milestone but could be implemented in future
+releases include:
+
+* Making the plugin dialog a side panel / side dockwidget (similar to VS Code
+  [^vscode-extensions-ui])
+* Allow for simultaneous install of plugins. Currently multiple plugins can be selected for
```

Comment:
+1

We're working on a "Collections" feature on the napari hub which will let users create and share arbitrary collections of plugins.

Some of the use cases that users are excited about are "getting started" collections for a specific domain or "suites" of plugins that work together well... all of which would benefit from a mechanism to install the full collection.

An improved mechanism for simultaneous installation of plugins would be high value for supporting this kind of a feature.

Reply:
If for whatever reason this becomes tricky, we can export those collections as conda metapackages too!